### DISCUSSION TITLE
Can't get accurate forces with the trained model.

### DISCUSSION THREAD

**Discussion by Lijieying** [deepmodeling/deepmd-kit#2914] [Q&A]
### Summary

The properties ones concerned is the accurate forces in research, almost 1E-3 to 1E-2 eV/Å. But the forces in training and validating converged to 1E-3 to 1E-2 eV/Å only, and the rmse of the predicted forces could only reach  1E-2 tp 1E-1 eV/Å. Is that normal?  If it is, will it mean different algorithoms of the force calculation between vasp and deepmd-kit? Or not, Is there anything wrong with my model training jobs?

### DeePMD-kit Version

2.1.5

### TensorFlow Version

2.9.0 (cpu version)

### Python Version, CUDA Version, GCC Version, LAMMPS Version, etc

python 3.10.0. 

### Details

The dataset for my model is generated from an equilibrium state with ab-init MD(npt,press from 0.0 to 1.0, temperature from 300 to 1200). And the accuracy of energy and forces are pretested, 1E-6 eV and 1E-4 eV/Å respectively. Training parameters are shown in attachements. the following lcurve.png is the best outcome of all my trained model.
Looking forward to your response. Thanks~
[input.json](https://github.com/deepmodeling/deepmd-kit/files/12877283/input.json)
![lcurve](https://github.com/deepmodeling/deepmd-kit/assets/73207509/9432a351-d36f-47f1-83f6-9dc8f7be326b)

**Comment by njzjz** [comment]
> But the forces in training and validating converged to 1E-3 to 1E-2 eV/Å only, and the rmse of the predicted forces could only reach 1E-2 tp 1E-1 eV/Å. Is that normal?

Could you explain this sentence? What is the difference between the training and validating forces and predicted forces?
